### ISSUE TITLE
IMAC. I can't open the game after several updates ago using Vulkan, now OpenGL is removed.

### ISSUE THREAD

**Issue by Bishop444** [ValveSoftware/Dota2-Gameplay#7181]
### Description

Since November 16, 2022, Dota does not run in IMAC, there will be a bug with the Vulkan engine, because on the recommendation of a user I tried using -GL (OpenGL) and it worked, now that that engine was removed the game cannot be opened, this problem It already exists MONTHS and I'm not the only one. Could you check? It's not the first time, I doubt it's the last.

**Comment by bryanCharlie** [comment]
I'm facing the same issue, the game is now unplayable.

**Comment by koreanfan** [comment]
Yeah, they remove opengl support from linux too. But with vulkan i got memory leak. After 2-3 matches i must reboot my pc to free my ram. I have 8gb ram on my main pc. Also on second pc with 16gb ram i have same issue, but i can play more mtaches and anyway i must reboot pc to free my ram.

**Comment by dchiang** [comment]
so Valve finally decided to screw us by completely dropping OpenGL support... I just played my first match after the last game update and didn't notice the game was running on Vulkan. Thanks Valve for the MMR, lost you are the best

**Comment by IbrahimFikry** [comment]
![Screenshot from 2023-03-07 17-35-48](https://user-images.githubusercontent.com/36134772/223385993-f5225004-ee28-41b7-ba8b-5edb48f423e3.png)

I got this error now. Is there any launch options that i can use to at least start the game?

**Comment by FanFanFantastic** [comment]
help, why is the game so dark on Vulkan

**Comment by Philaeux** [comment]
The OpenGL drop is not a surprised and was planned. As stated in the update 

```
As we mentioned when we launched [Battle Pass: Part II](https://steamcommunity.com/linkfilter/?url=https://www.dota2.com/newsentry/3398555399443921912), this update has removed support for OpenGL. For macOS customers, the minimum requirement is now macOS 10.13.6 with a Metal-capable GPU (AMD GCN-based GPU or newer; NVIDIA Kepler-based GPU or newer; or Intel HD 4000 or newer). For Linux customers, AMD GCN+, NVIDIA Kepler+, or Intel HD 5000-series or newer is required.
```

So make sure your hardware is supporting Vulkan, make sure you have your latest driver. 
If you have the required hardware and you experience some FPS issues in game, try to see if something is making it appear (like some hero, some spell, some particule effects) that could show that there is some issues on Valve's end.

**Comment by dchiang** [comment]
Yes, it was announced by Valve a long time ago, but the fact that the community had been replying and posting that Vulkan has performance issues in comparison to OpenGL from a long time ago also and that Valve still removed OpenGL support shows little consideration from Valve to the Dota2 community.

Nonetheless I have two setups, 1 with a Randeon R7 M440 (laptop) and the other one with an Intel HD Graphics 630 (Desktop). Both are newer and are suppose to perform better than the Intel HD Graphics 5000 according to benchmarks all over the internet. Both support a most recent version of Vulkan compared to the HD Graphics 5000, both of my setups use Ubuntu 20.04. I used to be able to have around 100~120 fps on both when using OpenGL and now I can hardly get over 45 fps and there are times when the fps drops around 20 or the screen completely freeze on both. There are also vsync issues and some other glitches when using Vulkan that valve hasn't fixed over the time but I consider fps drop to be the worst since it make the game almost not playable at all.
